pandas/core/apply.py

```diff
@@ -135,13 +141,14 @@ def __init__(
         ):
 
             def f(x):
+                assert callable(func)
                 return func(x, *args, **kwargs)
```

Comment:
Just an aside: we do this in several places; I've never understood what the benefit of currying is. I think things would be simpler if we just removed it (we wouldn't need orig_f).

Reply:
Yeah, agree.